tests/sanity/ignore-2.15.txt

```diff
@@ -1,3 +1,87 @@
+plugins/httpapi/dcnm.py import-3.10!skip
+plugins/httpapi/dcnm.py import-3.9!skip
+plugins/httpapi/dcnm.py validate-modules:missing-gplv3-license
+plugins/module_utils/common/models/ipv4_cidr_host.py import-3.10!skip
```

Comment:
We need to remove the sanity skips in order to certify the collection (here and the other skips)